lib/cbus_elixir/app/meeting.ex

```diff
@@ -1,16 +1,22 @@
 defmodule CbusElixir.App.Meeting do
   use Ecto.Schema
   import Ecto.Changeset
+  import Ecto.Query
   alias CbusElixir.App.Meeting
 
-
   schema "meetings" do
     field :date, :utc_datetime
     has_many :speakers, CbusElixir.App.Speaker
```

Comment:
`CbusElixir.App.Speaker` can be moved up to the top as an alias and this can be replaced with just `Speaker`